multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/WebMvcConfiguration.java

```diff
@@ -39,6 +40,9 @@ public void addInterceptors(InterceptorRegistry registry) {
     @Override
     public void configurePathMatch(PathMatchConfigurer configurer) {
         configurer.setUseSuffixPatternMatch(false);
+        UrlPathHelper urlPathHelper = new UrlPathHelper();
+        urlPathHelper.setUrlDecode(false);
```

Comment:
It should be check does handle correctly plus(+) signs in org/space names. We had previously another issues related with url decode/encode.

Reply:
It accepts + sign. This is covered by integration tests.